openerp/tools/translate.py

```diff
@@ -329,6 +329,8 @@ def next(self):
                 elif line.startswith('#,') and (line[2:].strip() == 'fuzzy'):
                     fuzzy = True
                 line = self.lines.pop(0).strip()
+            if 0 == len(self.lines):
```

Comment:
Well, I don't think so, because this is a better Python construction, but let's see what @xmo-odoo